### DISCUSSION TITLE
detected as junk virus 'Bundpil.Shortcut' by SMADAV

### DISCUSSION THREAD

**Discussion by 3xploiton3** [valinet/ExplorerPatcher#2028] [General]
https://i.imgur.com/CyK1Xd7.png

![image](https://github.com/valinet/ExplorerPatcher/assets/19517680/6010f6a9-0fd0-4b7f-ad20-53cb0e284c6a)

**Comment by Poopooracoocoo** [comment]
You'll need to report this false positive to the AV vendor
